sdk/src/main/java/io/dapr/client/ObjectSerializer.java

```diff
@@ -54,6 +56,11 @@ protected ObjectSerializer() {
       return (byte[]) state;
     }
 
+    // Proto buffer class is serialized directly.
+    if (state instanceof MessageLite) {
+      return ((MessageLite) state).toByteArray();
```

Comment:
This allows this to serialize proto objects.

Reply:
I think this is similar to how python handles gRPC messages too.

Reply:
Should his be documented somewhere?

Reply:
I will work on documentation later. This SDK actually does not have a nice doc website like the python SDK - yet.